src/engine/guidance/post_processing.cpp

```diff
@@ -1131,6 +1141,94 @@ std::vector<RouteStep> assignRelativeLocations(std::vector<RouteStep> steps,
     return steps;
 }
 
+std::vector<RouteStep> anticipateLaneChange(std::vector<RouteStep> steps)
+{
+    const constexpr auto MIN_DURATION_NEEDED_FOR_LANE_CHANGE = 15.;
+
+    // Postprocessing does not strictly guarantee for only turns
```

Comment:
Both New Name and Notification can have valid lane items that need to be handled. Why do restrict yourself here?